libraries/core-react/src/components/Select/Select.tokens.ts

```diff
@@ -62,4 +83,12 @@ export const multiSelect: ComponentToken = R.mergeDeepRight(select, {
     left: spacingMediumSmall,
     right: spacingLarge,
   },
+  modes: {
+    compact: {
+      spacings: {
+        top: '2px',
```

Comment:
Should we probably add `xx_small` (2px) here?

Reply:
Good point! Fixed 👌